### PR TITLE
Document new interface option to select light or dark theme

### PR DESCRIPTION
**Pull request by rdswift**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

Document option settings changed in https://github.com/metabrainz/picard/pull/1761

### Description of the Change

Remove outdated option and add description of new option.

### Additional Action Required

Please confirm the final note which states:

> If anything other than "Default" is selected, then the colors assigned in the Colors section will not be applied.

### CODE REVIEW

File: config/options_interface.rst [hunk 1]
```diff
@@ -58,6 +44,24 @@
...
+    On Linux and similar operating systems, the available choices are:
+
+    * Default - The default color scheme based on the operating system display settings.
+    * System - The Qt5 theme configured in the desktop environment.
+
+    .. note::
+
+        If anything other than "Default" is selected, then the colors assigned in the :doc:`options_interface_colors` section
+        will not be applied.
+
```
Comment (phw) [inline config/options_interface.rst:64]: That's actually not true, the colors are being used. There are even two sets of colors there, one for light and one for dark UI. Depending on how Picard is currently displayed one or another set will be available to edit.

Maybe a note that colors can be configured separately for light and dark UI cod be added to the color options.

**Review by phw — CHANGES_REQUESTED** [review]
Thanks. Looks good, only the note about interface colors needs to be changed.

File: config/options_interface.rst (hunk 1 above)
Comment (rdswift) [reply config/options_interface.rst:65]: Okay, I experimented a little and think I understand now.  I've changed the wording in the note on the interface page, and added a note to the interface colors page.

Comment (phw) [reply config/options_interface.rst:65]: Yes, that fits :) Thanks a lot

**Review by phw — APPROVED** [review]

**Pull request merged**